src/features/hotspots/root/HotspotsList.tsx

```diff
@@ -113,15 +127,31 @@ const HotspotsList = ({
   )
 
   return (
-    <BottomSheetSectionList
-      sections={sections}
-      keyExtractor={(item: Hotspot) => item.address}
-      ListHeaderComponent={<WelcomeOverview />}
-      renderSectionHeader={renderHeader}
-      renderItem={renderItem}
-      contentContainerStyle={contentContainerStyle}
-      showsVerticalScrollIndicator={false}
-    />
+    <Box backgroundColor="white" flex={1} top={visible ? 0 : wh}>
```

Comment:
The iOS native status bar time and service text is in dark mode (so that its white) on this screen, but when you have the list open you can no longer see this text. I think it should be in normal mode (black text) instead?

Reply:
I noticed this too but forgot to make a change. I'd like it to adjust per tab. I'll figure it out...